lib/linguist/generated.rb

```diff
@@ -525,5 +526,18 @@ def generated_grpc_cpp?
       return false unless lines.count > 1
       return lines[0].start_with?("// Generated by the gRPC")
     end
+    
+    # Internal: Is this an owl-generated Dart file?
+    #
+    # An own generated file contains:
+    # // GENERATED CODE - DO NOT MODIFY
+    # on the first line.
+    #
+    # Return true or false
+    def generated_dart?
+      return false unless extname == '.dart'
+      return false unless lines.count > 1
+      return lines[0].include?("GENERATED CODE - DO NOT MODIFY")
```

Comment:
Looks like `//  Generated code. Do not modify.` [is common too](https://github.com/search?p=1&q=extension%3Adart+%22GENERATED+CODE%22+%22DO+NOT+MODIFY%22&type=Code&utf8=%E2%9C%93). Maybe add a second case for it?